app/api/log/formatMessage.js

```diff
@@ -1,7 +1,17 @@
+import { tenants } from 'api/tenants';
+import { config } from 'api/config';
+
 export default function formatMessage(info, instanceName) {
   const message = info.message && info.message.join ? info.message.join('\n') : info.message;
 
-  const result = `${info.timestamp} [${instanceName}] ${message}`;
+  const tenant = tenants.current();
+  let tenantName = tenant.name;
```

Comment:
Maybe refactor to a ternary conditional?

Reply:
done 👍